shuup/utils/locale/fi_FI/LC_MESSAGES/django.po

```diff
@@ -91,6 +92,10 @@ msgstr[1] "%s päivää"
 msgid "%(min)s--%(max)s days"
 msgstr "%(min)s--%(max)s päivää"
 
+#, python-format
+msgid "Node %s not in depth-first order"
+msgstr "Solmu %s ei ole järjestyksessä"
```

Comment:
This is crazy, humm :P 

Maybe: `Yhtymäkohta %s ei ole "syvyys ensin" -järjestyksessä`